.github/workflows/ci.yml

```
@@ -22,28 +22,27 @@ jobs:
         run: |
           python -m pip install --disable-pip-version-check .
       - name: Run tests on ${{ matrix.os }}
-        run: nox --non-interactive --session "tests-${{ matrix.python-version }}" -- --full-trace
+        run: nox --non-interactive --session "tests-${{ matrix.python-version }}" "cover" -- --full-trace
```

Should no longer be necessary:

```suggestion
        run: nox --non-interactive --session "tests-${{ matrix.python-version }}"  -- --full-trace
```

Same below in the `build-py310` job.

---

I thought this too but when running `nox -s tests` locally it wouldn't notify `cover`. I'll see if it works in GHA.

---

Ahh brilliant, seems to have worked! 👍🏻